### DISCUSSION TITLE
`aarch64-linux-android-ar` not installed

### DISCUSSION THREAD

**Discussion by mendess** [termux/termux-packages#8214] [Q&A] [answered]
I'm trying to compile a rust program but it fails to build one of the dependencies with the error: `error occurred: Failed to find tool. Is 'aarch64-linux-android-ar' installed? `

I've been searching for a while but I can't find a package that gives me this binary, does anyone know?

**Comment by thunder-coding** [comment, marked as answer]
Try installing the `binutils` package, or use `AR=llvm-ar <your cargo build command>`

**Comment by irediaes** [comment]
To fix copy and rename `llvm-ar` to `aarch64-linux-android-ar` in `${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/{SYSTEM}/bin`

Do this for all your target arch.

**Comment by finagolfin** [comment]
We should probably patch the rust package to use llvm-ar instead, as I did with the Swift package. I don't use rust, so will have to be someone who uses it.
